configman/config_manager.py

```diff
@@ -461,7 +461,10 @@ def write_conf(self, config_file_type, opener, skip_keys=None):
                 an_option = option_defs[a_key]
                 if ((not a_key.startswith('admin'))
                     and isinstance(an_option, Option)
-                    and an_option.secret
+                    and (
+                        an_option.secret
+                        or 'password' in an_option.name.lower()
+                    )
```

Comment:
you know, the other solution is to don't automatically protect anything.  Make the change in the socorro to explicitly protect the fields that we want protected.  I'm thinking I like that better, because it keeps the surprise factor out of configman.